core/scripts/go.mod

```diff
@@ -379,3 +379,5 @@ replace (
 	// replicating the replace directive on cosmos SDK
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 )
+
+exclude github.com/sourcegraph/sourcegraph/lib v0.0.0-20221216004406-749998a2ac74
```

Comment:
See https://github.com/smartcontractkit/chainlink/pull/14446, I merged to my local one so that I could get the IDE working.



That PR should merge before mine so the diff won't show up.